### PR TITLE
chore(cmd): change table label 

### LINKED ISSUE #268: Rename the table name this label to action, what do you think?
I was thinking to move this label to action, what do you think?

_Originally posted by @vincenzopalazzo in https://github.com/coffee-tools/coffee/pull/239#discussion_r1563962790_

### PR DESCRIPTION
**Pull request by Anyitechs**
This PR addresses @tareknaser and @vincenzopalazzo suggestion in [#239 (comment](https://github.com/coffee-tools/coffee/pull/239#discussion_r1563209333))

Closes #268

### CODE REVIEW

**Comment by vincenzopalazzo** [comment]
Coffee has the best documentation that I ever written, please read the commit style https://coffee-docs.netlify.app/contributing-to-coffee

File: coffee_cmd/src/coffee_term/command_show.rs [hunk 1]
```diff
@@ -97,7 +97,7 @@ pub fn show_nurse_verify(nurse_verify: &ChainOfResponsibilityStatus) -> Result<(
         table.push([
             term::format::dim(String::from("●")),
             term::format::bold(String::from("Defects")),
-            term::format::bold(String::from("Affected repositories")),
+            term::format::bold(String::from("Action")),
```
[the remark this answers is not in the document]
Comment (vincenzopalazzo) [reply coffee_cmd/src/coffee_term/command_show.rs:100]: `Affected` sounds a really good one

Comment (vincenzopalazzo) [inline coffee_cmd/src/coffee_term/command_show.rs:100]:
```suggestion
            term::format::bold(String::from("Affected")),
```

thanks @tareknaser for the suggestion

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
One thing is hard in coding, and it is naming things 😄 

After that ping me and we merge the PR

**Comment by Anyitechs** [comment]
> Coffee has the best documentation that I ever written, please read the commit style https://coffee-docs.netlify.app/contributing-to-coffee

Thank you @vincenzopalazzo for sharing the link. Recent commits are following the style guide now.

**Review by tareknaser — APPROVED** [review]

**Pull request merged**